libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -656,7 +677,9 @@ void AP_MotorsMulticopter::output_logic()
 
         // set and increment ramp variables
         _spin_up_ratio = 1.0f;
-        _throttle_thrust_max -= spool_step;
+        const float spool_time = _spool_down_time > minimum_spool_time ? _spool_down_time : _spool_up_time;
+        const float spool_adjustment = -1.0f / (spool_time * _dt);
+        _throttle_thrust_max += spool_adjustment;
```

Comment:
I like:
_throttle_thrust_max -= spool_step;

As it is clear things are being reduced. Is there a reason to change this?

Reply:
There isn't anymore. At an earlier form I had kept `spool_step` as precomputed, and was avoiding aliasing, but at this point we aren't doing that anymore, so I've swapped it back.